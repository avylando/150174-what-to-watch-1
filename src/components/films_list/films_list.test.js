import React from "react";
import renderer from "react-test-renderer";
import FilmsList from "./films_list.jsx";

it(`FilmsList correctly renders FilmCards`, () => {
  const tree = renderer
    .create(
        <FilmsList
          filmsData={[
            {
              title: `Bohemian Rhapsody`,
              imageSrc: `img/bohemian-rhapsody.jpg`,
            },
            {
              title: `Macbeth`,
              imageSrc: `img/macbeth.jpg`,
            },
            {
              title: `Pulp Fiction`,
              imageSrc: `img/pulp-fiction.jpg`,
            }
          ]}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
