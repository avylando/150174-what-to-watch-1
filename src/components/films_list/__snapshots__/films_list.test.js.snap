// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsList correctly renders FilmCards 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Bohemian Rhapsody"
        height="175"
        src="img/bohemian-rhapsody.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Macbeth"
        height="175"
        src="img/macbeth.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Macbeth
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
  >
    <button
      className="small-movie-card__play-btn"
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Pulp Fiction"
        height="175"
        src="img/pulp-fiction.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Pulp Fiction
      </a>
    </h3>
  </article>
</div>
`;
